src/ir/dot/dot_generator.cc

```diff
@@ -136,6 +139,7 @@ std::string DotGeneratorHelper::GetDotNode(const Operation& op) {
   //
   // The `Ix` and `Rx` are ports that we can connect edges to.
   // (cf. https://www.graphviz.org/doc/info/shapes.html#record)
+  static absl::btree_set<std::string> empty_results_set;
```

Comment:
Good point. I will make it const. I don't know if we can make it constexpr. This is mainly needed to support the `operation_labeler` interface. Once we have the results directly in operation, this will go away. However, it is needed for now.

Reply:
Fixed this using a recommended idiom.